AtReconstruction/AtCopyAuxTreeTask.h

```diff
@@ -0,0 +1,77 @@
+#ifndef ATCOPYAUXTREETASK_H
+#define ATCOPYAUXTREETASK_H
+/*
+ * Task to copy the tree structure of an auxilary input root file to an output root file
+ * when using AtDataReductionTask with ATTPC files.
+ *
+ * An example would be when sorting ATTPC events with an associated root file from
+ * a separate DAQ system (such as HiRAEVT). This task would copy only the
+ * corresponding events to a new output file, ensuring parity between the root files
+ * from the two DAQs.
+ *
+ */
+
+#include <TChain.h>
+#include <TString.h>
+#include <TTree.h>
+// FairRoot classes
+#include <FairTask.h>
```

Comment:
/user/anthonya/attpcroot/AtReconstruction/AtCopyAuxTreeTask.h should remove these lines:
- #include <TTree.h>  // lines 16-16
- #include <vector>  // lines 23-23